tls/s2n_record_write.c

```diff
@@ -153,15 +211,13 @@ int s2n_record_write(struct s2n_connection *conn, uint8_t content_type, struct s
     }
 
     /* Compute non-payload parts of the MAC(seq num, type, proto vers, fragment length) for composite ciphers.
-     * Composite "encrypt" will MAC the payload data and filling in padding.
-     */
+     * Composite "encrypt" will MAC the payload data and filling in padding.*/
```

Comment:
I know this wasn't your original change, but should this be "fill" instead of "filling"?